src/ui/components/Header/Header.tsx

```diff
@@ -122,6 +122,11 @@ function HeaderTitle({
     }
   }, [editing, hasTitle, recording.title]);
 
+  const testName = recording.metadata?.test?.title;
```

Comment:
Let's not do this. I don't like how this introduces a different concept of title based on what kind of replay you're looking at.

I agree that `Replay of devtools--3vku99qej` is **not** what you want to see on that upper left side. But we should fix that on the test adapter side — it should have been saving the title on these replays as "Replay of <insert-test-name-here>".

Reply:
Happy to remove this IF statement when it is obsolete, but for now i think we should use this work around